src/pages/dataElements/form/CustomAttributes.tsx

```diff
@@ -64,9 +64,8 @@ function CustomAttribute({ attribute, index }: CustomAttributeProps) {
         )
     }
 
-    throw new Error(
-        `@TODO(CustomAttributes): Implement value type "${attribute.valueType}"!`
-    )
+    // @TODO: Verify that all value types have been covered!
+    throw new Error(`Implement value type "${attribute.valueType}"!`)
 }
 
 export function CustomAttributes({
```

Comment:
Related to CustomAttributes; can we add a `No value` option to attributes that are optionsets? Currently there's no way to deselect after an option is selected.